lib/install/management/inspect.go

```diff
@@ -147,7 +147,7 @@ func (d *Dispatcher) GetDockerAPICommand(conf *config.VirtualContainerHostConfig
 			}
 
 			dEnv = append(dEnv, "DOCKER_TLS_VERIFY=1")
-			info, err := os.Stat(conf.ExecutorConfig.Name)
+			info, err := os.Stat(certpath)
```

Comment:
That's a good question. What is the correct output if they're in different locations?